apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -770,3 +804,39 @@ function EditTotalScorePercForm({ resLocals, id }: { resLocals: Record<string, a
     </form>
   `;
 }
+
+function ResetQuestionVariantsModal({
+  csrfToken,
+  groupWork,
+}: {
+  csrfToken: string;
+  groupWork: boolean;
+}) {
+  return Modal({
+    id: 'resetQuestionVariantsModal',
+    title: 'Confirm reset question variants',
+    body: html`
+      <p>
+        Are your sure you want to reset all current variants of this question for this
+        ${groupWork ? 'group' : 'student'}?
+        <strong>All ungraded attempts will be lost.</strong>
+      </p>
+      <p>
+        This ${groupWork ? 'group' : 'student'} will receive a new variant the next time they view
+        this question.
+      </p>
+    `,
+    footer: html`
+      <input type="hidden" name="__action" value="reset_question_variants" />
+      <input type="hidden" name="__csrf_token" value="${csrfToken}" />
+      <input
+        type="hidden"
+        name="unsafe_instance_question_id"
+        class="js-instance-question-id"
+        value=""
```

Comment:
Nit: don't bother setting the empty string explicitly. (I know this was copied from @nwalters512's code)